src/main/paradox/features/akka-cluster-bootstrap.md

```diff
@@ -0,0 +1,44 @@
+## Akka Cluster Bootstrap
+
+When enabled, this feature will use service discovery to identify other contact points. Once the requisite number of
+contact points have been identified, the one with the lowest address will form the cluster. The other nodes will then join that one.
+
+This feature is automatically enabled for Lagom applications that require it. It can be manually enabled with the following build configuration:
+
+```sbt
+enableAkkaClusterBootstrap := true
+```
+
+When you generate resources for a deployment, you'll need to specify a replica count greater than 2. A value of 3 or
+5 is recommended. This is to reduce the risk of multiple islands of clusters being formed when first formed. For example,
+the following command specifies 5 replicas which are used as the contact points for forming the cluster:
```

Comment:
The wording is a bit confusing here. Is this conflating the minimum number of nodes bootstrap expects with the number of how many replicas we start? Is it automagically set to the same number?

Reply:
Not sure. Let's open an issue for it and table that for this PR.

Reply:
https://github.com/lightbend/lightbend-orchestration-docs/issues/55